### DISCUSSION TITLE
"Commenting out" extra fields

### DISCUSSION THREAD

**Discussion by zackbatist** [retorquere/zotero-better-bibtex#2979] [Q&A]
There are a few instances when I need to use [extra fields](https://retorque.re/zotero-better-bibtex/exporting/extra-fields/index.html#bbt-specific), but I would like the ability to toggle them on or off. Is there a convention or leading character that signals my desire to ignore a specific line?

**Comment by retorquere** [comment]
A postscript can do that.

**Reply by zackbatist** [reply]
From what I gather, it's necessary to encode the specific fields you want to toggle on/off in the script. Is it possible to use this to write a more generic function to toggle whatever field on the line that the script is activated for?

**Reply by retorquere** [reply]
I don't understand what you're asking here, sorry. If you submit a debug log ID for the item you're looking at (right-click, BBT, send debug log) and post here what you want the output to look like I'll be better able to help you.

**Reply by zackbatist** [reply]
Something akin to `%` in latex, `#` in R or `<!--> x </--!>` in html.

A specific use case: Sometimes I want to use different variations of `shortauthor` depending on the context of the thing I'm writing (especially relevant when the author is an organization or institution, which can be abbreviated in different ways, as per the record in log K95WNH45-refs-euc/6.7.227-7). Rather than having to re-write different values each time, I could simply have more than one `tex.shortauthor` recorded on separate lines. I would then switch between them by prefixing a character to prompt the translator to ignore all character that follow it until the end of the line.

This would also be extremely useful for documenting my rationale behind using an extra field, the context of use, or references to a website that documents its use.
